deepmd/pt/model/model/sezm_model.py

```
@@ -1524,14 +1709,103 @@ def trace_and_compile(
+            vals: tuple[torch.Tensor, ...],
+        ) -> dict[str, torch.Tensor | None]:
+            """Temporarily replace model buffers/attrs with FX proxy tensors.
+
+            Executed at trace time inside compute_fn.  make_fx records the
+            proxy tensors as placeholder nodes, so the compiled graph reads them
+            as live inputs rather than baked-in constants.  The ``finally``
+            block in compute_fn always calls ``_restore_task_bufs`` to leave
+            the model in its original state after tracing.
+            """
+            saved: dict[str, torch.Tensor | None] = {}
+            for name, val in zip(task_buf_names, vals):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)